### PR TITLE
Add scenario to ensure that uptodate state works properly on recurring actions

### PR DESCRIPTION
**Pull request by wweellddeerr**
## What does this PR change?

See title.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: testsuite

- [x] **DONE**

## Test coverage
- Cucumber tests were added

- [x] **DONE**

## Links

Fixes https://github.com/SUSE/spacewalk/issues/21488

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by cbbayburt** [comment]
@ktsamis FYI

File: testsuite/features/step_definitions/common_steps.rb [hunk 1]
```diff
@@ -201,6 +201,11 @@
   raise "#{arg1} can't be checked" unless choose(arg1)
 end
 
+When(/^I check default base channel radio button of this "([^"]*)"$/) do |host|
+  arg = BASE_CHANNEL_BY_CLIENT[host]
```
Comment (srbarrios) [inline testsuite/features/step_definitions/common_steps.rb:205]: Use a more meaningful name for that variable please.

**Review by srbarrios — APPROVED** [review]
LGTM.
But please before merging it, be sure the test pass in the Pull Request tests: https://ci.suse.de/view/Manager/view/Uyuni-PRs/job/uyuni-prs-ci-tests/

**Review by cbbayburt — APPROVED** [review]
LGTM :+1: thanks, @wweellddeerr

**Pull request merged**